mycore-base/src/main/java/org/mycore/common/xml/MCRURIResolver.java

```diff
@@ -155,6 +154,11 @@ public final class MCRURIResolver implements URIResolver {
      * Creates a new MCRURIResolver
      */
     private MCRURIResolver() {
+        reInit();
```

Comment:
move reInit() to static {} block and remove constructor.

Reply:
is fixed